rosidl_typesupport_connext_cpp/resource/msg__type_support.cpp.em

```diff
@@ -12,31 +13,17 @@
 @#  - get_header_filename_from_msg_name (function)
 @#######################################################################
 @
+#include "@(spec.base_type.pkg_name)/@(subfolder)/dds_connext/@(get_header_filename_from_msg_name(spec.base_type.type))__type_support.hpp"
+
 #include <limits>
 #include <stdexcept>
 
-#include <rosidl_generator_c/message_type_support_struct.h>
+#include "rosidl_generator_c/message_type_support_struct.h"
 // this is defined in the rosidl_typesupport_connext_cpp package and
 // is in the include/rosidl_typesupport_connext_cpp/impl folder
-#include <rosidl_generator_cpp/message_type_support.hpp>
-#include <rosidl_typesupport_connext_cpp/visibility_control.h>
-
-#include "@(spec.base_type.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.base_type.type))__struct.hpp"
-#ifndef _WIN32
-# pragma GCC diagnostic push
-# pragma GCC diagnostic ignored "-Wunused-parameter"
-# pragma GCC diagnostic ignored "-Wunused-variable"
-# ifdef __clang__
-#  pragma clang diagnostic ignored "-Wdeprecated-register"
-#  pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
-# endif
-#endif
-#include "@(spec.base_type.pkg_name)/@(subfolder)/dds_connext/@(spec.base_type.type)_Support.h"
-#ifndef _WIN32
-# pragma GCC diagnostic pop
-#endif
+#include "rosidl_generator_cpp/message_type_support.hpp"
+#include "rosidl_typesupport_connext_cpp/visibility_control.h"
```

Comment:
Included here as user header while already included as system header ([here](https://github.com/ros2/rosidl/blob/5dd82983849bf37df60d1a6f61890b810ebb1ad0/rosidl_typesupport_introspection_cpp/resource/srv__type_support.cpp.em#L19))
Same below for services